### PR TITLE
Cleanup ia32cfg module

### PR DESCRIPTION
**Pull request by BrentHoltsclaw**
Cleanup of imports
Enabled check per thread

Signed-off-by: BrentHoltsclaw <brent.holtsclaw@intel.com>

### CODE REVIEW

File: chipsec/modules/common/ia32cfg.py [hunk 1]
```diff
@@ -43,23 +43,22 @@ def check_ia32feature_control(self):
         self.logger.start_test( "IA32 Feature Control Lock" )
         self.logger.log( "[*] Verifying IA32_Feature_Control MSR is locked on all logical CPUs.." )
 
-        ok = True
+        ok = ModuleResult.PASSED
         for tid in range(self.cs.msr.get_cpu_thread_count()):
-            #feature_cntl = chipsec.chipset.read_register( self.cs, 'IA32_FEATURE_CONTROL', tid )
-            #chipsec.chipset.print_register( self.cs, 'IA32_FEATURE_CONTROL', feature_cntl )
+            feature_cntl = self.cs.read_register( 'IA32_FEATURE_CONTROL', tid )
```
Comment (frinzell) [inline chipsec/modules/common/ia32cfg.py]: This is unused unless in `VERBOSE` mode

File: chipsec/modules/common/ia32cfg.py [hunk 2]
```diff
@@ -43,23 +43,24 @@ def check_ia32feature_control(self):
...
-            feature_cntl_lock = self.cs.get_control('Ia32FeatureControlLock' )
-            self.logger.log( "[*] cpu%d: IA32_Feature_Control Lock = %d" % (tid,feature_cntl_lock) )
-            if 0 == feature_cntl_lock: ok = False
+            if self.logger.VERBOSE:
+                feature_cntl = self.cs.read_register( 'IA32_FEATURE_CONTROL', tid )
+                self.cs.print_register( self.cs, 'IA32_FEATURE_CONTROL', feature_cntl )
+            feature_cntl_lock = self.cs.get_control('Ia32FeatureControlLock', tid )
+            self.logger.log( "[*] cpu{:d}: IA32_Feature_Control Lock = {:d}".format(tid,feature_cntl_lock) )
+            if 0 == feature_cntl_lock:
+                ok = ModuleResult.FAILED
```
Comment (ErikBjorge) [inline chipsec/modules/common/ia32cfg.py]: We should rename the variable 'ok' as this implies a Boolean value.  Something like result would be better now that we are passing ModuleResult around in it.

File: chipsec/chipset.py [hunk 3]
```diff
@@ -991,7 +991,7 @@ def get_control(self, control_name, cpu_thread=0, with_print=0):
         control = self.Cfg.CONTROLS[ control_name ]
         reg     = control['register']
         field   = control['field']
-        reg_data = self.read_register(reg)
+        reg_data = self.read_register(reg,cpu_thread)
```
Comment (ErikBjorge) [inline chipsec/chipset.py]: A space after the comma would be nice.

File: chipsec/modules/common/ia32cfg.py [hunk 4]
```diff
@@ -43,23 +43,24 @@ def check_ia32feature_control(self):
...
+                self.cs.print_register( self.cs, 'IA32_FEATURE_CONTROL', feature_cntl )
+            feature_cntl_lock = self.cs.get_control('Ia32FeatureControlLock', tid )
             self.logger.log( "[*] cpu{:d}: IA32_Feature_Control Lock = {:d}".format(tid,feature_cntl_lock) )
-            if 0 == feature_cntl_lock: ok = False
+            if 0 == feature_cntl_lock:
+                res = ModuleResult.FAILED
 
-        if ok:
-           self.res = ModuleResult.PASSED
+        if RecursionError == ModuleResult.PASSED:
```
Comment (NicholasArmour) [inline chipsec/modules/common/ia32cfg.py]: RecursionError should be res?

**Pull request merged**